payments/models.py

```diff
@@ -105,46 +81,95 @@ class Meta:
         abstract = True
 
 
-class BerthProduct(AbstractPlaceProduct):
-    price_group = models.ForeignKey(
-        BerthPriceGroup,
-        on_delete=models.CASCADE,
-        verbose_name=_("price group"),
-        related_name="products",
+class BerthProductManager(models.Manager):
+    def get_in_range(self, width: Union[Decimal, float, int]):
+        return (
+            self.get_queryset()
+            .filter(min_width__lt=width, max_width__gte=width)
+            .first()
```

Comment:
Hmm good point. It _shouldn't_ happen, but if that were the case, it would take the best guess. But if that happens, then we have bigger problems than selecting the correct product. 😅